src/openfermion/utils/_operator_utils.py

```diff
@@ -32,6 +32,19 @@ class OperatorUtilsError(Exception):
     pass
 
 
+def inline_sum(summands, seed):
```

Comment:
This argument order is way better. It would be even nicer if the seeds defaulted to additive and multiplicative identities IMO!

Reply:
Actually wait, is there a nice way to do this including both Fermion and QubitOperators?

Reply:
You could get one of the summands and use the class method. That wouldn't be nice, though, because summands is an iterable and not a sequence.

Reply:
Yeah, oh well, I'll approve